### PR TITLE
Added WPF (net47) platform

### PR DESCRIPTION
**Pull request by tkouba**
### Added WPF platform ###

.Net Framework 4.7 WPF platform support

### Issues Resolved ### 
- #764

### API Changes ###
 None

### Platforms Affected ### 
- WPF

### Behavioral Changes ###
None

### Testing Procedure ###
Added new project Samples.Wpf for testing

### PR Checklist ###

- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

**Comment by aritchie** [comment]
I really appreciate the PR.  The only thing that is needed is the removal of the xamarin forms reference.

**Comment by tkouba** [comment]
Reference Xamarin.Forms.Platform.WPF is needed for platform support. This implementation uses Xamarin.Forms.Platform.WPF.Controls.FormsContentDialog for showing dialogs. Dialogs are not WPF windows, but it is shown as control and overlays only application window.

File: src/Acr.UserDialogs/Acr.UserDialogs.csproj [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <LangVersion>latest</LangVersion>
         <NoWarn>$(NoWarn);1591;1701;1591;1702;1705;VSX1000;NU1603</NoWarn>
         <DisableExtraReferences>true</DisableExtraReferences>
-        <GenerateLibraryLayout Condition="$(TargetFramework.StartsWith('uap'))">true</GenerateLibraryLayout>
+        <GenerateLibraryLayout Condition=" $(TargetFramework.StartsWith('uap')) OR $(TargetFramework.StartsWith('net7')) ">true</GenerateLibraryLayout>
```
Comment (VladislavAntonyuk) [inline src/Acr.UserDialogs/Acr.UserDialogs.csproj:19]: .NET 7 is not released yet. I suppose you should replace it with net47

Comment (tkouba) [reply src/Acr.UserDialogs/Acr.UserDialogs.csproj:19]: Only net4 for forward compatibility

Comment (VladislavAntonyuk) [reply src/Acr.UserDialogs/Acr.UserDialogs.csproj:19]: ok, but not net7

Comment (tkouba) [reply src/Acr.UserDialogs/Acr.UserDialogs.csproj]: Fixed, thanks

**Comment by aritchie** [comment]
I have decided not to take this PR for a couple of reasons.

1) Main one - I won't take a dependency on Xamarin Forms.  The whole point to this library was to be abstract from XF, so the native Xamarin apps could use it.

2) I don't use WPF professionally at all anymore.  I don't want to support this going forward. 

I do want to thank you for the PR though.  I would suggest releasing it as a secondary library though.

**Comment by tkouba** [comment]
I understand the reasons, I will try to prepare the secondary library as an extension.

**Pull request closed without merging**